Frontend/src/lib/pages/UsersPage.svelte

```diff
@@ -24,11 +22,25 @@
 
     let colors = ["red", "blue", "green", "pink"];
 
+    (function() {
```

Comment:
I would use Svelte's onMount instead of raw dogging a function like that

Reply:
This entire function was for the ContextMenu. It is currently very different than this and is on the CID-9 feature branch
[https://github.com/WallochMatt/CertifID/tree/feature-CID-9](url)